components/src/MultilineInput/MultilineInput.js

```diff
@@ -0,0 +1,75 @@
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { transparentize } from "polished";
+import { space } from "styled-system";
+import theme from "../theme";
+import { subPx } from "../Utils";
+
+const MultilineInputStyles = {
+  disabled: {
+    color: transparentize(0.67, theme.colors.black),
+    borderColor: theme.colors.lightGrey,
+    backgroundColor: theme.colors.whiteGrey,
+  },
+  error: {
+    color: theme.colors.red,
+    borderColor: theme.colors.red,
+  },
+  default: {
+    color: theme.colors.black,
+    borderColor: theme.colors.grey,
+  },
+};
+
+const getMultilineInputStyle = props => {
+  if (props.disabled) { return MultilineInputStyles.disabled; }
+  if (props.error) { return MultilineInputStyles.error; }
+  return MultilineInputStyles.default;
+};
+
+const MultilineInput = styled.textarea.attrs(({ error, required, placeholder }) => ({
```

Comment:
Why is it called `MultilineInput` as opposed to `TextArea`?

Reply:
Inspired by https://polaris.shopify.com/components/forms/text-field. I though that this was very elegant and logical solution.

Reply:
my unsolicited opinion: textarea has the benefit of being html default, but multiline input is more descriptive. i'd approve of either, prefering multilineinput for its descriptiveness.

Reply:
Renamed MultilineInput to Textarea. We can revisit this decision when we revisit Input API next week.